stdlib/char.gr

```diff
@@ -1,35 +1,56 @@
+/**
+ * @module Char: Utilities for working with chars.
+ * A Grain `Char` represents a single [Unicode scalar value](https://www.unicode.org/glossary/#unicode_scalar_value)
+ * @example import Char from "char"
+ * 
+ * @since 0.3.0
+ */
+
 import WasmI32 from "runtime/unsafe/wasmi32"
 import Memory from "runtime/unsafe/memory"
 import Errors from "runtime/unsafe/errors"
 import { tagSimpleNumber, allocateChar, allocateString } from "runtime/dataStructures"
 
-export *
-
 exception MalformedUtf8
 
-// The minimum value of Unicode characters
-// Number
-let min = 0x0000
-
-// The maximum value of Unicode characters
-// Number
-let max = 0x10FFFF
-
-// Returns true if the given number is a valid Unicode scalar value.
-// @param n: Number - the value to check
-// @returns Bool
-let isValid = (n) => {
-  n >= min && (n <= 0xD7FF || n >= 0xE000) && n <= max
+/**
+ * The minimum valid Unicode charCode.
```

Comment:
```suggestion
 * The minimum valid Unicode scalar value.
```
We should either do "Unicode scalar value" or "Unicode Scalar Value". @ospencer what is your preference?

Reply:
Lowercase!